package_info.json

```diff
@@ -0,0 +1,4 @@
+{
+  "package_name": "th2-data-services-rdp",
+  "package_version": "0.0.0.1"
```

Comment:
Let's call it by data-sources naming convention (https://github.com/th2-net/th2-ds-source-lwdp/tree/master?tab=readme-ov-file#23-release-versioning)  -> 5.0.0.1